### DISCUSSION TITLE
Plain ARM UEFI bootloader without L4T-Launcher

### DISCUSSION THREAD

**Discussion by mickeyproc** [OE4T/meta-tegra#1545] [Q&A]
Is it possible to use a plain ARM UEFI bootloader without L4T-Launcher to boot the nvidia target (Orin Nano in my case)?
I'm not very bootloader knowledgable, but the whole nvidia bootloader seems a a bit complicated and with caveats as explained [here](https://github.com/OE4T/meta-tegra/wiki/extlinux.conf-support#caveats).

It would be very nice to just have a (somewhat) plain UEFI that can be customized with variables (for RAUC integration for instance) and be able to boot a rootfs partition.

Is this something anyone has thought of and maybe investigated? - If not, do you have any pointers on how to get started with this?

**Comment by dwalkes** [comment]
> Is this something anyone has thought of and maybe investigated?

The closest is probably the work Matt did to look at systemd-boot, referenced [in several of the previous meeting notes](https://github.com/search?q=repo%3AOE4T%2Fmeta-tegra+systemd-boot&type=wikis).  

That or the reference to Grub as an anternative loader in the [35.4.1 release notes](https://developer.nvidia.com/embedded/jetpack-sdk-512) or the thread you've previously started [here](https://github.com/OE4T/meta-tegra/discussions/1480)

**Reply by mickeyproc** [reply]
Oh right, completely forgot I had created that discussion. Closing this as a duplicate to #1480
